mvg/utils/response_processing.py

```diff
@@ -0,0 +1,57 @@
+"""
+This module contains helper functions for manipulating the response from endpoints.
+"""
+
+from enum import Enum
+from typing import Dict, Union, Callable
+
+
+class SortOrder(Enum):
+    ASC = "asc"
+    DESC = "desc"
+
+
+def get_paginated_items(
+    request: Callable, url: str, params: Dict[str, Union[int, SortOrder]]
```

Comment:
Nice helper method. But I am just wondering if this method can be reused easily for `get_analysis_results` in `MVGAPI` since very soon we will have pagination setup for the `get_analysis_results` in the backend.

Reply:
I don't see any reason for why it shouldn't work, as long as the targeted endpoint (_/analyses/requests/{request_id}/results_ in this case) implements the offset pagination functionality.

Reply:
Sure